server/views/docs/visual-architecture.config.js

```diff
@@ -1,5 +1,5 @@
-import breakpoints from 'wellcomecollection-config/breakpoints';
-import gridConfig from 'wellcomecollection-config/grid-config';
+import breakpoints from '../../ui-conf/breakpoints';
```

Comment:
could we call it `ui-config`? I don't think we use `conf` anywhere else.

Reply:
Done